### PR TITLE
replace master with main branch & license fix

### PR DESCRIPTION
**Pull request by mzitnik**

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -18,10 +18,12 @@ on:
     branches:
       - "master"
       - "branch-*"
+      - "main"
```
Comment (pan3793) [inline .github/workflows/build-and-test.yml:21]: why not just replace "master" with "main", instead of appending?

File: LICENSE [hunk 2]
```diff
@@ -1,3 +1,5 @@
+Copyright 2016-2024 ClickHouse, Inc.
```
Comment (pan3793) [inline LICENSE:1]: this is not the right place to add the copyright, this file should be kept as-is. 

usually, copyright info should be added to each file's license header, and the NOTICE file.

File: LICENSE [hunk 3]
```diff
@@ -186,7 +188,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2016-2024 ClickHouse, Inc.
```
Comment (pan3793) [inline LICENSE:191]: ditto, should be kept as-is, you can refer to some ASF projects.

File: README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Spark ClickHouse Connector
 ===
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.github.housepower/clickhouse-spark-runtime-3.3_2.12/badge.svg)](https://search.maven.org/search?q=clickhouse-spark-runtime)
-[![License](https://img.shields.io/github/license/housepower/spark-clickhouse-connector)](https://github.com/housepower/spark-clickhouse-connector/blob/master/LICENSE)
+[![License](https://img.shields.io/github/license/housepower/spark-clickhouse-connector)](https://github.com/housepower/spark-clickhouse-connector/blob/main/LICENSE)
```
Comment (pan3793) [inline README.md:4]: org name should be replaced

File: .github/workflows/build-and-test.yml (hunk 1 above)
Comment (mzitnik) [reply .github/workflows/build-and-test.yml:21]: I was thinking after we merge to master rename the branch to main and also add protection to the brach (so eventually master will be removed)

File: LICENSE (hunk 2 above)
Comment (mzitnik) [reply LICENSE:1]: This our convention in ClickHouse i will check with legal

File: LICENSE (hunk 3 above)
Comment (mzitnik) [reply LICENSE:191]: same here as i have mention on the previous comment

**Review by mshustov — APPROVED** [review]

File: .github/workflows/deploy-website.yml [hunk 5]
```diff
@@ -17,6 +17,7 @@ on:
   push:
     branches:
       - "master"
+      - "main"
```
Comment (pan3793) [inline .github/workflows/deploy-website.yml:20]: seems the ClickHouse org does not enable the GitHub Pages, you may need to find out another place to host the docs.

Comment (mzitnik) [reply .github/workflows/deploy-website.yml:20]: We will probably move it to ClickHouse docs 
Just to verify you are referring to the website

Comment (pan3793) [reply .github/workflows/deploy-website.yml:20]: yes, the website, the previous link is https://housepower.github.io/spark-clickhouse-connector, is not available after the transfer.

**Pull request merged**